src/Umbraco.Web.BackOffice/Controllers/EntityController.cs

```diff
@@ -745,7 +748,14 @@ public IEnumerable<EntityBasic> GetChildren(int id, UmbracoEntityTypes type, Gui
             {
                 return new PagedResult<EntityBasic>(0, 0, 0);
             }
-
+            //adding multiple conditions ,considering id,key & name as filter param
+            var clauses = new List<Expression<Func<IUmbracoEntity, bool>>>();
+            int.TryParse(filter, out int filterAsIntId);
+            Guid.TryParse(filter, out Guid filterAsGuid);
+            clauses.Add(x =>
```

Comment:
Hi @dKumarmj ,
Thanks for fixing this, it's working great! 👍
I have one question about using a "clauses" collection: there seem to always be only one element in it, so I thought it might be simpler to just build a filter string?
Or maybe else put each clause of the "||" query in a separate clause, so that the "WhereAny" statement below makes it more performant?
What do you think?

Reply:
ALso, I forgot to mention that it seems that the file package-lock.json file was probably added in the commit by accident 😉

Cheers!

Reply:
@mikecp  yes, thanks for pointing out this, yes we don't need to use a clause condition in this case, now I changed to simple `||` condition, please see the latest commit here https://github.com/umbraco/Umbraco-CMS/pull/13735/commits/c059f181a59bb087648478cad9e5d119e43e78c0

Yes, please ignore the `src/Umbraco.Web.UI.Client/package-lock.json` change 😉

Reply:
Hi @dKumarmj ,

Thanks for the quick update! I removed the package-lock file, so it's ready to get merged.

Thanks again for spotting and solving the issue 👍

Cheers!